.eslintrc.json

```
@@ -0,0 +1,15 @@
+{
+    "env": {
+        "browser": true,
```

```suggestion
        "node": true,
```

for now I think we should target the node env instead of the browser so we can access node level vars and other configs

read more [here](https://eslint.org/docs/latest/use/configure/language-options)

---

Yes for sure, I agree will change that now!